api/cmd/build/clean.go

```diff
@@ -124,8 +123,9 @@ func clean() {
 	for tag := range tags {
 		args := []string{"rmi", tag}
 		cmd = manifest.Docker(args...)
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-		cmd.Run()
+		_, err := cmd.CombinedOutput()
+		if err != nil {
+			fmt.Printf("WARNING: %s\n", err)
```

Comment:
I've noticed with the docker cli, the output can have information about the error. Printing `err` sometimes results in `exit status 125`.